desktop/cmp/form/RadioInput.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {PropTypes as PT} from 'prop-types';
+import {isObject} from 'lodash';
+import {HoistComponent, elemFactory} from '@xh/hoist/core';
+import {observable, action} from '@xh/hoist/mobx';
+import {radio, radioGroup} from '@xh/hoist/kit/blueprint';
+import {withDefault} from '@xh/hoist/utils/js';
+
+import {HoistInput} from '@xh/hoist/cmp/form';
+import './RadioInput.scss';
+
+
```

Comment:
Note: as it stands if you tab into the radio group you can select a button using the arrow keys, but the enter key will not choose the highlighted options. Should we change that?